### ISSUE TITLE
[Documentation issue] What metric will be calculated if I use metrics=['accuracy'] ?

### ISSUE THREAD

**Issue by mario98** [keras-team/keras#8692]
What metric will be calculated if I use `metrics=['accuracy']` or even `metrics=['acc']` in model compilation, as seen in many keras examples? In [metrics.py](https://github.com/fchollet/keras/blob/0bc8fac4463c68faa3b3c415c26eab02aa361fd5/keras/metrics.py) there is no such thing as 'accuracy' or 'acc'. Even worse, it is not explained in [any documentation](https://keras.io/search.html?q=metric). This should be improved.
